src/vs/platform/list/browser/listService.ts

```diff
@@ -1480,7 +1480,7 @@ configurationRegistry.registerConfiguration({
 		},
 		[treeStickyScroll]: {
 			type: 'boolean',
-			default: product.quality !== 'stable',
+			default: typeof product.quality === 'string' && product.quality !== 'stable', // only enable as default in insiders
```

Comment:
You might want it to be enabled in OSS as well

Reply:
I agree, but I think that is the case here, as I am check that the quality is not set to 'stable'. The comment is misleading a bit

Reply:
I thought maybe `quality` is undefined in OSS but I can't remember how it works off the top of my head